source/extensions/filters/listener/original_dst/original_dst.cc

```diff
@@ -27,6 +27,7 @@ Network::FilterStatus OriginalDstFilter::onAccept(Network::ListenerFilterCallbac
     // the address returned by getOriginalDst() matches the local address of the new socket.
     // In this case the listener handles the connection directly and does not hand it off.
     if (original_local_address) {
+      ENVOY_LOG(trace, "original_dst: set destination to %s", original_local_address->ip()->addressAsString());
```

Comment:
Why not `original_local_address->asString()`, which include the port also.

Reply:
This also feel like a right log place for Windows since there is failure path for Windows , you better log at https://github.com/envoyproxy/envoy/pull/20565/files#diff-97891057e3641865b911786d7407ec01fa1cbe2e5ffb20b263834ec5ac69e4a6R72

Reply:
Which line do you want to point?

Reply:
sorry, before the line 72

Reply:
Thanks.